newlib/libc/machine/microblaze/strcpy.c

```diff
@@ -90,7 +90,7 @@ strcpy (char *__restrict dst0,
 #if defined(PREFER_SIZE_OVER_SPEED) || defined(__OPTIMIZE_SIZE__)
   char *s = dst0;
 
-  while (*dst0++ = *src0++)
+  while ((*dst0++ = *src0++))
```

Comment:
thanks for the cleanup

Reply:
14k files being compiled. 5k x 3 lines of warnings of `hey you should put that into a parentheses` :D
Another one was `-fstack-protector is not supported` warning. Took care of it too.